dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -32,13 +32,46 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
 
+/**
+ * Option for {@link StaticLinkageChecker}. To construct an input class path, the checker requires
+ * either one of the following types of input:
+ *
+ * <ul>
+ *   <li>{@code bomCoordinate}: a Maven coordinate for a BOM
```

Comment:
a Maven coordinate --> Maven coordinates

Reply:
Maven docs supports the singular:

https://maven.apache.org/pom.html#Maven_Coordinates

> **groupId, artifactId, version:**
> You will see these elements often. This trinity is used to compute the Maven coordinate of a specific project in time, demarcating it as a dependency of this project.

Reply:
Elliotte and I checked the statement yesterday but that's the only document (I could find) that uses singular. Maybe it means

> This trinity is used to compute _(each of)_ the Maven coordinate of a specific project: _groupId, artifactId, and version_".

Maven Aether library uses plural for coordinates.
http://maven.apache.org/resolver/apidocs/org/eclipse/aether/artifact/DefaultArtifact.html#DefaultArtifact(java.lang.String)

Reply:
Sad times for our pluralization options...